server/app/views/applicant/ProgramIndexView.java

```diff
@@ -129,14 +132,9 @@ private DivTag topContent(String titleText, String infoTextLine1, String infoTex
                 "px-6",
                 StyleUtils.responsiveSmall("mb-6"));
 
-    DivTag infoLine1Div =
+    DivTag infoDiv =
```

Comment:
**nitpick:** It looks like the mocks have this text taking up a smaller width so it shows on two lines instead of one. Would you mind updating the styling to do that or check with Miriam if this is okay?

Reply:
Done.